File: FrontEnd/src/models/workflowHistoryDTO.ts
export interface WorkflowHistoryDTO {
  id: string;
  idWorkflowInstance: string;
  idUser: string;
  action: string | null;
  idUnit: string;
  createdDate: Date;
  status: number | null;
  isStepCompleted: boolean;
  comment: string | null;
  message: string | null;
  isCancelled: boolean;
  isRequestToChanged: boolean;
}

File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/WorkflowHistoryDto.cs
namespace DPD.HumanResources.Dtos.Dto;

public class WorkflowHistoryDto
{
    public Guid Id {get;set;}
    public Guid IdWorkFlowInstance {get;set;}
    public Guid IdUser {get;set;}
    public string? Action {get;set;}
    public Guid IdUnit {get;set;}
    public DateTime? CreatedDate {get;set;}
    public int? Status {get;set;}
    public bool IsStepCompleted {get;set;}
    public bool IsCancelled {get;set;}
    public bool IsRequestToChanged {get;set;}
    public string? Comment {get;set;}
    public string? Message {get;set;}
}